modules/post/multi/gather/minio_client.rb

```diff
@@ -0,0 +1,121 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Gather MinIO Client Key',
+        'Description' => %q{
+          This is a module that searches for MinIO Client credentials on a windows remote host.
+        },
+        'License' => MSF_LICENSE,
+        'References' => [
+          [ 'URL', 'https://blog.kali-team.cn/Metasploit-MinIO-Client-7d940c60ae8545aeaa29c96536dda855' ]
+        ],
+        'Author' => ['Kali-Team <kali-team[at]qq.com>'],
+        'Platform' => [ 'win', 'linux', 'osx' ],
```

Comment:
Since shell sessions are supported, you should add `unix` platform as well. This is how shell payloads are defined on linux/unix.
```suggestion
        'Platform' => [ 'win', 'linux', 'osx', 'unix' ],
```